presto-native-execution/README.md

```diff
@@ -96,17 +85,90 @@ checks.  When we need one it can be added.
 
 ## Development Env.
 
-### Setting up on macOs
-
-See `scripts/setup-macos.sh`
-
-After running the setup script add the cmake-format bin to your $PATH, maybe
-something like this in your ~/.profile:
+### Basic setting up on MacOs
 
+* Clone the whole Presto repository.
+* Run `presto/presto-native-execution/scripts/setup-macos.sh`.
+* Add the cmake-format bin to your $PATH, maybe something like this
+  in your ~/.profile:
 ```
 export PATH=$HOME/bin:$HOME/Library/Python/3.7/bin:$PATH
 ```
+* It is recommended to use some sort of IDE on Mac. CLion is an option.
 
 ### Setting up on Linux (CentOs 8 or later)
 
-See `scripts/setup-linux.sh`
+* Clone the whole Presto repository.
+* Run `presto/presto-native-execution/scripts/setup-linux.sh`
+
+### Setting up local Presto environment on MacOs (with optional IntelliJ and CLion) 
+Clone the whole Presto repository.
+
+Close IntelliJ and CLion
+
+In local Presto repo
+* `git fetch upstream`
+* `git co upstream/master`
+* `mvn clean install -DskipTests -T1C -pl -presto-docs`
+
+Run IntelliJ
+* Edit/Create `HiveQueryRunnerExternal` Run/Debug Configuration (alter paths accordingly).
+  * Main class: `com.facebook.presto.hive.HiveExternalWorkerQueryRunner`.
+  * VM options: `-ea -Xmx5G -XX:+ExitOnOutOfMemoryError -Duser.timezone=America/Bahia_Banderas -Dhive.security=legacy`.
+  * Working directory: `$MODULE_DIR$`
+  * Environment variables: `PRESTO_SERVER=/Users/spershin/git/presto/presto-native-execution/cmake-build-debug/presto_cpp/main/presto_server;DATA_DIR=/Users/spershin/Desktop/data;WORKER_COUNT=0`
+  * Use classpath of module: choose `presto-native-execution` module.
+* Edit create `TestHiveQueriesUsingJSON` Test Run/Debug Configuration (alter paths accordingly).
+  * Class: `com.facebook.presto.nativeworker.TestHiveQueriesUsingJSON`
+  * VM Options: `-ea -DPRESTO_SERVER=/Users/spershin/git/presto_cpp/cmake-build-debug/presto_cpp/main/presto_server -DDATA_DIR=/Users/spershin/Desktop/data`
```

Comment:
Should we replace 'spershin' with something more generic?

Reply:
Well, it does say "(alter paths accordingly)" above. :)
But I can put <user> instead.